src/webhook/webhook-receiver.ts

```diff
@@ -0,0 +1,80 @@
+import { Request, Response } from "express";
+import { verify } from "@octokit/webhooks-methods";
+import { GitHubServerApp } from "../models/git-hub-server-app";
+import { getPayload } from "./get-payload";
+import { Webhooks } from "./webhooks";
+import { getLogger } from "../config/logger";
+import { WebhookContext } from "./types";
+
+const WEBHOOK_HEADERS = [
+	"x-github-event",
+	"x-hub-signature-256",
+	"x-github-delivery",
+];
+
+export const webhookReceiver = async (webhooks: Webhooks, request: Request, response: Response) => {
+	const missingHeaders = (WEBHOOK_HEADERS.filter((header) => !(header in request.headers))).join(", ");
```

Comment:
there's a better way to do this kind of validation at the route level using `express-validator`

Reply:
Now using `express-validator` to check for headers